homework13/homework13.py

```diff
@@ -0,0 +1,203 @@
+# Exercise 1 phonebook
+import json
+
+
+# MyCustomException
+class MyCustomException(Exception):
+    def __init__(self, text):
+        self.txt = text
+
+
+# My phone book
+# contacts = []
+
+FORMAT_STR = '{:<15} {:>12}'
+
+
+def stats():
+    with open('homework13/phonebook.txt') as file:
+        file_content = file.read()
+        if len(file_content) == "":
+            print("Phone book is empty")
+        else:
+            contacts = json.loads(file_content)
+            count = len(contacts)
+            print(f"There are {count} entries in the phone book")
+
+
+def add():
+    with open('homework13/phonebook.txt') as file:
+        file_content = file.read()
+        if file_content == "":
```

Comment:
nit: too many line duplicates, so it would be better to move it out to own function and for writing to file too